suite/tests/linux/syscall_pwait.cpp

```diff
@@ -415,6 +415,40 @@ main(int argc, char *argv[])
 
     execute_subtest(main_thread, &test_set, psyscall_ppoll, true);
 
+    auto psyscall_raw_epoll_pwait = [test_set](bool nullsigmask) -> int {
+        int epoll_fd = epoll_create1(EPOLL_CLOEXEC);
+        struct epoll_event events;
+        return syscall(SYS_epoll_pwait, epoll_fd, &events, 24, 60000, 0);
+    };
+
+    auto psyscall_raw_pselect = [test_set](bool nullsigmask) -> int {
```

Comment:
This isn't really a NULL *sigmask*: it's a NULL *struct* holding the 6th and 7th args.  Best to test both.

Reply:
Added, done.